OsuPlayer/Windows/ExportSongsProcessWindow.axaml.cs

```diff
@@ -74,29 +76,68 @@ internal async Task ExportSongs(CancellationToken token)
 
                 var hashLength = mapEntry.Hash.Length < 8 ? mapEntry.Hash.Length : 8;
 
-                var fileName = $"{mapEntry.GetArtist()} - {mapEntry.GetTitle()} ({mapEntry.Hash.Substring(0, hashLength)}).mp3";
+                var fileName = $"{mapEntry.Artist} - {mapEntry.Title} ({mapEntry.Hash.Substring(0, hashLength)}).mp3";
 
                 fileName = string.Join("_", fileName.Split(Path.GetInvalidFileNameChars(), StringSplitOptions.RemoveEmptyEntries));
 
                 var exportPath = Path.Combine(_path, fileName);
 
                 try
                 {
-                    #region Copy song to export folder asynchroniously
-
                     const FileOptions fileOptions = FileOptions.Asynchronous | FileOptions.SequentialScan;
                     const int bufferSize = 81920;
 
-                    await using (var sourceStream = new FileStream(mapEntry.FullPath, FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize, fileOptions))
+                    if (mapEntry.FullPath.EndsWith(".mp3"))
```

Comment:
I know this sounds stupid, but make the statement the following `mapEntry.FullPath.ToLower().EndsWith(".mp3")` as it will also include weird files like .MP3 and stuff. unless `EndsWith()` is case insensitive

Reply:
It says outdated, because I reviewed your one commit, as the second one is just copied to own method. Change still stays there tho

Reply:
Changed